admin/metrics/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='OSFStatistic',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('users', models.IntegerField(default=0, verbose_name=b'OSF users')),
+                ('delta_users', models.IntegerField(default=0)),
+                ('unregistered_users', models.IntegerField(default=0, verbose_name=b'Unregistered users')),
+                ('projects', models.IntegerField(default=0, verbose_name=b'Number of projects')),
+                ('delta_projects', models.IntegerField(default=0)),
+                ('public_projects', models.IntegerField(default=0, verbose_name=b'Number of public projects')),
+                ('delta_public_projects', models.IntegerField(default=0)),
+                ('registered_projects', models.IntegerField(default=0, verbose_name=b'Number of projects registered')),
+                ('delta_registered_projects', models.IntegerField(default=0)),
+                ('date', models.DateTimeField(default=None)),
```

Comment:
How is this list supposed to relate to the one at https://github.com/CenterForOpenScience/osf.io/blob/db2cc04fbc9f11fb3148754419c32af2eb954308/scripts/analytics/benchmarks.py#L120-L135?